Source/Core/DolphinQt/Config/Mapping/IOWindow.cpp

```diff
@@ -207,7 +207,7 @@ class InputStateLineEdit : public QLineEdit
 
 private:
   std::function<ControlState()> m_state_evaluator;
-  bool m_should_paint_state_indicator;
+  bool m_should_paint_state_indicator = true;
```

Comment:
Shouldn't it be initialised to false?

Reply:
I'm not sure.  I based it on this code:

https://github.com/dolphin-emu/dolphin/blob/4b8b53ac732645f6c0fd3f4b11733cb3d86c9c41/Source/Core/DolphinQt/Config/Mapping/IOWindow.cpp#L670-L689

which sets it to `true` in the default case.  But maybe `false` would be better since the default should be empty.